includes/gateways/class-wpsms-gateway-tskin.php

```
@@ -0,0 +1,77 @@
+    public $unit;
+    public $flash = 'disabled';
+    public $isflash = false;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->bulk_send      = false;
+        $this->has_key        = true;
+        $this->validateNumber = __('Enter the recipient number with its country code and no leading plus sign, for example 966500000000.', 'wp-sms');
+        $this->documentUrl    = 'https://app.tskin.sa/wpplugin/assets/api-docs.php';
```

PHPStan (2.2.7)

[error] Property WP_SMS\Gateway::$documentUrl (bool) does not accept string.


(assign.propertyType)